core/templates/dev/head/css/oppia.css

```diff
@@ -1055,7 +1055,7 @@ textarea {
   background: linear-gradient(to right, rgba(0,0,0,0.65) 0%, rgba(0,0,0,0) 100%);
   /*The display is set in Library.js*/
   display: none;
-  height: 240px;
+  height: 259px;
```

Comment:
Btw there's a lot of duplication between this and the next class. Think it's better to collect the common bits in an

```
.oppia-library-carousel-overlay-left, .oppia-library-carousel-overlay-right {
   ...
}
```

selector?